### PR TITLE
Update design for version info modal

### PR DESCRIPTION
**Pull request by Avi98**
### Component/Part

Frontend

### Description
This PR improves the design of version info modal

### Steps

- [x] Added implementation
- [ ] Added / updated tests
- [ ] Added / updated documentation
- [ ] Added changelog snippet
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

File: frontend/locales/en.json [hunk 1]
```diff
@@ -157,7 +157,7 @@
       "issueTracker": "Found a bug? Fill an issue!",
       "sourceCode": "Read the source code",
       "versionInfo": "Version info",
-      "title": "You are using"
+      "title": "Your username"
```
Comment (mrdrogdrog) [inline frontend/locales/en.json:160]: ???

File: frontend/src/components/landing-layout/footer/version-info/version-info-modal.tsx [hunk 2]
```diff
@@ -40,27 +40,27 @@ export const VersionInfoModal: React.FC<CommonModalProps> = ({ onHide, show }) =
...
-            i18nKey={'landing.versionInfo.issueTracker'}
-            className={'btn btn-sm btn-primary d-block mb-2'}
-            href={links.issues}
-          />
-        </ShowIf>
+        <div className='d-flex justify-content-between mt-3'>
+          <ShowIf condition={!!links.sourceCode}>
+            <TranslatedExternalLink
+              i18nKey={'landing.versionInfo.sourceCode'}
+              className={'btn btn-sm btn-primary d-block mb-2'}
```
Comment (mrdrogdrog) [inline frontend/src/components/landing-layout/footer/version-info/version-info-modal.tsx:52]: Remove the btn-sm class from both buttons.

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: frontend/locales/en.json (hunk 1 above)
Comment (Avi98) [reply frontend/locales/en.json:160]: Should only change the title height

Comment (ErikMichelson) [reply frontend/locales/en.json:160]: Could it be that you confused the meaning of this field?

The dialog should show the version of HedgeDoc you're currently using, for example "2.0.0". However, this version is retrieved from the backend and therefore not available in the mock-mode. To indicate this (and to not have a blank field), the dialog shows "mock" in this case.

But: This has nothing to do with the user nor its name. That the username in mock-mode is "mock" too is just coincidence.

File: frontend/src/components/landing-layout/footer/version-info/version-info-modal.tsx (hunk 2 above)
Comment (Avi98) [reply frontend/src/components/landing-layout/footer/version-info/version-info-modal.tsx]: done

File: frontend/locales/en.json (hunk 1 above)
Comment (Avi98) [reply frontend/locales/en.json]: Thanks a lot @ErikMichelson  for making it clear for me. With real server we get the correct version number. 

Screenshot 2023-04-20 at 4 14 57 PM

Comment (mrdrogdrog) [reply frontend/locales/en.json]: Please change the translation to something that makes a bit more sense. Like "Running HedgeDoc Version"

Comment (Avi98) [reply frontend/locales/en.json]: done

**Review by mrdrogdrog — APPROVED** [review]

**Comment by mrdrogdrog** [comment]
image

**Review by DerMolly — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**